fragment_betriebssysteme.tex

```diff
@@ -692,27 +698,27 @@ \section{Betriebssysteme - Grundlagen}
 
 \question{Wie ist eine Platte intern organisiert? Wie wirkt sich dies auf den Informationszugriff aus? Wie geht das UNIX Fast File System damit um?}
 \begin{answer}
-  Eine physikalische Festplatte besteht aus sechs bis zehn internen Platten (Scheiben). Auf jede
-  dieser internen Platten greift jeweils von oben und von unten ein Lese-Schreibkopf zu. Jede
-  Oberfläche ist in Spuren unterteilt, jede Spur in Sektoren. übereinanderliegende Spuren auf den
-  verschiedenen Oberflächen bilden Zylinder. Da für jede Armbewegung relativ viel Zeit benötigt
+  Eine physikalische Festplatte besteht aus sechs bis zehn internen \crucial{Platten} (Scheiben). Auf jede
+  dieser internen Platten greift jeweils von oben und von unten ein \crucial{Lese-Schreibkopf} zu. Jede
+  Oberfläche ist in \crucial{Spuren} unterteilt, jede Spur in \crucial{Sektoren}. übereinanderliegende Spuren auf den
+  verschiedenen Oberflächen bilden \crucial{Zylinder}. Da für jede Armbewegung relativ viel Zeit benötigt
   wird, sollten Armbewegungen möglichst vermieden werden. D.h. zusammengehörende Daten sollten
   in hintereinanderliegenden Sektoren in derselben Spur untergebracht werden, damit das Lesen
   und Schreiben in Vorgang ohne weitere Armbewegung vorgenommen werden kann. Ist die Datenmenge
   größer, sollten die Spuren innerhalb des gleichen Zylinders verwendet werden.
 
-  % FIXME
+  % FIXME % why?
```

Comment:
No Idea. It's been ages and I probably fixed but forgot to remove the comment. Probably best to remove it if you could do that in a --fixup commit.